returnn/tf/layers/base.py

```diff
@@ -1344,6 +1344,14 @@ class InternalLayer(LayerBase):
   """
 
 
+class DataNotAvailableLayer(InternalLayer):
+  """
+  This is dummy layer that is created when the output template is "not available for inference"
+  The output
+
+  """
```

Comment:
Incomplete documentation?

Reply:
Ah yes, it seems I stashed a part of it away.